packages/@react-types/checkbox/src/index.d.ts

```diff
@@ -82,7 +82,13 @@ export interface SpectrumCheckboxProps extends AriaCheckboxProps, StyleProps {
   /**
    * This prop sets the emphasized style which provides visual prominence.
    */
-  isEmphasized?: boolean
+  isEmphasized?: boolean,
+  /**
+   * A slot name for the component. Slots allow the component to receive props from a parent component.
+   * An explicit `null` value indicates that the local props completely override all props received from a parent.
+   * @private
+   */
+  slot?: string | null
```

Comment:
Needed so RSP Checkbox can receive the `slot="selection"` props from RAC Tree context